.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -86,7 +97,7 @@ body:
       required: true
   - type: textarea
     attributes:
-      label: What happened?
+      label: What happened? Describe the command as it is despite any usage of alias.
```

Comment:
I think you might want to use not `label` but `value`.

Reply:
`value` is way smaller... OK